### ISSUE TITLE
Parameter setting and selection of variables

### ISSUE THREAD

**Issue by graduatefast** [198808xc/Pangu-Weather#59]
Thanks for your invaluable work. I'm trying to train the models based on your pseudocode. However, I met some questions, and I will appreciate if you can provide some information:
1. As you mentioned, the four models with different lead time steps are trained by using the same algorithms. Are the parameter setting of the training of these four models are totally same? I'm also wondering what the training loss values are after the 100 epochs.
2. If all the variables (both in pressure levels and surface level) are necessary for training the model? If I only use surface variables, will the model performance will be greatly reduced?

Thanks very much

**Comment by graduatefast** [comment]
By the way, the area I selected is only around 15° * 20°, not the global area. Appreciate if you could kind provide some information.

**Comment by 198808xc** [comment]
Hi,

To train good weather forecasting models, both the global data (rather than a region of 15 degs x 20 degs or so) is needed. This is because the future weather largely depends on the outer region -- imagine a winter storm that propagates from west to east. Without seeing weather data from the outer region, it is impossible for the model to "predict" when the storm will come to the region you are interested in.

Also, both upper-air and surface variables are needed. The reason is similar -- it is difficult to predict the surface variables alone without seeing the upper-air variables.

So, I think your training strategy needs to be changed, without which the observed loss values are less meaningful. BTW, we used the same hyper-parameters to train all four models.
